dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -571,6 +614,25 @@ class DistriOptimizer[T: ClassTag] (
 
     model
   }
+
+  private def getLatestFile(fileName: String): String = {
+    val fl = new java.io.File(checkpointPath.get)
+    val files = fl.listFiles(new FilenameFilter {
+      override def accept(dir: File, name: String): Boolean = {
+        name.startsWith(fileName)
+      }
+    })
+
+    var lastMod = Long.MinValue
```

Comment:
I think we don't need worry about this if they are not running in the same time as we can distinguish model files by time. If user are running more than one model at the same time and set the same checkpoint directory, what about if we create a sub directory with UUID to save the model files.